hyperparameter_tuning/model_tuning_for_distributed_training/hyperparameter_tuning_for_distributed_training.ipynb

```diff
@@ -0,0 +1,1372 @@
+{
```

Comment:
<p>Python 3.7 and below EOL in two months - can you make sure this notebook runs on Data Science 3.0 kernel instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3854/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>